public/framework/redux/store/index.ts

```diff
@@ -38,12 +38,12 @@ const store = configureStore({
 
 if (process.env.NODE_ENV === 'development' && module.hot) {
   module.hot.accept('./rootReducer', () => {
-    store.replaceReducer(persistReducer(persistConfig, require('./rootReducer').default));
+    store.replaceReducer(persistReducer(persistConfig, require('./rootReducer').default)); // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
Will change this line to conform liniting rules, just wanted to keep this PR with fewer unrelated changes.